### PR TITLE
MINOR: PartitionInfo toString corrected

### PR DESCRIPTION
**Pull request by kamalcph**
- Removed the extra ',' character while printing the replicas / in-sync replicas identifier

### CODE REVIEW

**Comment by ijuma** [comment]
Thanks for the PR. Can you please add a unit test? Since the original bug eluded us, it indicates that a simple test that validates the output would be useful.

File: clients/src/main/java/org/apache/kafka/common/PartitionInfo.java [hunk 1]
```diff
@@ -80,13 +80,10 @@ public String toString() {
     /* Extract the node ids from each item in the array and format for display */
     private String fmtNodeIds(Node[] nodes) {
```
Comment (ijuma) [inline clients/src/main/java/org/apache/kafka/common/PartitionInfo.java]: Can you please rename this to `formatNodeIds`. The name is unnecessarily cryptic (unrelated to this PR, I know).

**Comment by kamalcph** [comment]
@ijuma

Updated the method name and unit test added.

**Comment by ijuma** [comment]
LGTM, thanks. Merged to trunk.

**Pull request closed without merging**